accepted/2025/dotnet-pack-nuspec.md

```diff
@@ -0,0 +1,122 @@
+This proposal aims to add support for .nuspec-only packaging in dotnet pack, eliminating the need for a .csproj file and improving developer experience across platforms.
+The two main options for implementation are: (1) allowing `.nuspec` as a direct argument to `dotnet pack`, and (2) introducing a new subcommand such as `dotnet pack nuspec`.
+
+## Motivation
+
+Customers have been requesting the functionality of packing using a .nuspec file when using the command `dotnet pack` in files or non-SDK-style projects.
+
+## Proposed Approach: [\<PROJECT\>|\<SOLUTION\>|\<NUSPEC\>]
+
+The recommended approach is to allow `dotnet pack` to accept a .nuspec file as an argument, enabling the command to pack files without requiring a .csproj file.
+This would allow users to run `dotnet pack <nuspec>` directly, without needing to create a dummy .csproj file.
+
+## Explanation
+
+### Functional explanation
+
+Currently, arguments for `dotnet pack` are [\<PROJECT\>|\<SOLUTION\>].
+This feature will enable customers to also use a .nuspec to pack files.
+`dotnet pack` is available for projects that are using the SDK-style format.
+Adding the ability to use a .nuspec path instead will enable customers to pack projects or files that aren't SDK-style.
+In these cases, `dotnet pack` with a .nuspec will not run `NuGet.Build.Tasks.Pack.targets` and will instead use the current implementation of `nuget.exe` [pack commands](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.CommandLine/Commands/PackCommand.cs).
+This runs `PackCommandRunner`, which manages the packaging process in the `dotnet pack` command.
+It collects inputs, processes them, and generates NuGet packages.
+This component supports packing from .nuspec files, enhancing cross-platform workflows and eliminating the need for a .csproj file.
+
+### Technical explanation
+
+Currently, dotnet pack options and nuget.exe pack options are very different, meaning that we will need to add new options that would not apply when packing an SDK project.
+Some options, such as `--property` (or `-p:`), are already supported in MSBuild, `dotnet pack`, and `nuget.exe pack` for setting build properties.
+For .nuspec-only support, `--property` will be reused and interpreted for token replacement in the `.nuspec` file, similar to how `nuget.exe pack -Properties` works.
+Other options, such as `--output`, are already standard in the dotnet CLI and will be reused for .nuspec-only support, with clarified behavior where necessary.
+All new options and existing ones will need to be checked to see if they are applicable to the user scenario.
+This implementation does not require a new command or subcommand, making the command look cleaner and ensuring customers won't need to learn a new command.
+This is the ideal implementation.
+
+#### Example: Packing a .nuspec File with `dotnet pack`
```

Comment:
doc feedback: It'd argue this is functional documentation. 
The commands and how they work and all that is functional and tells the end user how to use the command.